bug: one failing unit test

### Describe the bug

One unit test is failing on our CI

See full logs [here](https://prow2.kubestellar.io/view/s3/prow-logs/pr-logs/pull/kubestellar_kubeflex/563/pull-kubestellar-kubeflex-test-unit/1980702372875210752)
```
go fmt ./...
api/v1alpha1/conditions_test.go
internal/controller/controlplane_controller.go
pkg/reconcilers/shared/postcreate_hook.go
pkg/reconcilers/shared/reconciler.go
pkg/util/status_check.go
go vet ./...
go: downloading github.com/onsi/ginkgo/v2 v2.22.2
go: downloading github.com/onsi/gomega v1.36.2
test -s /home/prow/go/src/github.com/kubestellar/kubeflex/bin/setup-envtest || GOBIN=/home/prow/go/src/github.com/kubestellar/kubeflex/bin go install sigs.k8s.io/controller-runtime/tools/setup-envtest@latest
go: downloading sigs.k8s.io/controller-runtime/tools/setup-envtest v0.0.0-20251020210837-fb2beabd029b
go: downloading sigs.k8s.io/controller-runtime v0.22.3
go: sigs.k8s.io/controller-runtime/tools/setup-envtest@latest: sigs.k8s.io/controller-runtime/tools/setup-envtest@v0.0.0-20251020210837-fb2beabd029b requires go >= 1.25.0 (running go 1.24.2; GOTOOLCHAIN=local)
make: *** [Makefile:249: /home/prow/go/src/github.com/kubestellar/kubeflex/bin/setup-envtest] Error 1 
```

### Steps To Reproduce

1. Create a PR
2. Wait for the unittesting pipeline to be triggered

### Expected Behavior

All unit tests must be green

### Additional Context

Looks like a dependency has been upgraded and is no longer in sync with go version.

--- a/Makefile
+++ b/Makefile
@@ -246,7 +246,7 @@ $(CONTROLLER_GEN): $(LOCALBIN)
 .PHONY: envtest
 envtest: $(ENVTEST) ## Download envtest-setup locally if necessary.
 $(ENVTEST): $(LOCALBIN)
-	test -s $(LOCALBIN)/setup-envtest || GOBIN=$(LOCALBIN) go install sigs.k8s.io/controller-runtime/tools/setup-envtest@release-0.22
+	test -s $(LOCALBIN)/setup-envtest || GOBIN=$(LOCALBIN) go install sigs.k8s.io/controller-runtime/tools/setup-envtest@release-0.19
 
 .PHONY: verify-go-versions
 verify-go-versions:

